langkit/support/langkit_support-diagnostics-output.ads

```diff
@@ -0,0 +1,37 @@
+package Langkit_Support.Diagnostics.Output is
```

Comment:
Please add copyright headers in both files. Also, could you add a package-level docstring?